src/hooks/useEmptyViewHeaderHeight/index.ios.ts

```diff
@@ -1,10 +1,11 @@
 import useSafeAreaInsets from '@hooks/useSafeAreaInsets';
 import {BUTTON_HEIGHT, BUTTON_MARGIN, HEADER_HEIGHT} from './const';
 
-function useEmptyViewHeaderHeight(isSmallScreenWidth: boolean): number {
+function useEmptyViewHeaderHeight(isSmallScreenWidth: boolean, areHeaderButtonsDisplayed: boolean): number {
```

Comment:
```suggestion
function useEmptyViewHeaderHeight(isSmallScreenWidth: boolean, isHeaderButtonsDisplayed: boolean): number {
```

Reply:
<img width="1054" alt="Screenshot 2024-10-30 at 12 31 13" src="https://github.com/user-attachments/assets/38b98bbd-ecea-4c58-9791-0c7d1ccc5002">
 here we also have plural, nobody protested 😄

Reply:
another examples 😃 
<img width="1200" alt="Screenshot 2024-10-30 at 12 32 24" src="https://github.com/user-attachments/assets/98619b45-8a45-425e-b8f3-3a94feb4ffde">